src/model/model.nit

```diff
@@ -627,7 +638,7 @@ class MClassDef
 	var in_hierarchy: nullable POSetElement[MClassDef] = null
 
 	# Is the definition the one that introduced `mclass`?
-	fun is_intro: Bool do return mclass.intro == self
+	fun is_intro: Bool do return isset mclass._intro and mclass.intro == self
```

Comment:
I do not like the old style id with a _. If one have a better proposal for the low-level attribute access...